flowersteam/vivarium
# Logging system

## Description
Replaced `print` statements with python logging system

## Related Issue (if applicable)
#9 

## How to Test
Launch the server
```
python3 vivarium/simulator/grpc_server/simulator_server.py
```
Launch the Panel interface
```
panel serve vivarium/interface/panel_app.py --autoreload
```

You should see in the terminal that all the info printed have the level of importance and the file the info comes from, such as:
```
INFO:__main__:Simulator server started
```
